pkg/logs/tailers/windowsevent/tailer_test.go

```diff
@@ -23,8 +24,8 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/logs/message"
 	"github.com/DataDog/datadog-agent/pkg/logs/sources"
 	"github.com/DataDog/datadog-agent/pkg/util/testutil/flake"
-	"github.com/DataDog/datadog-agent/pkg/util/winutil/eventlog/api"
-	"github.com/DataDog/datadog-agent/pkg/util/winutil/eventlog/test"
+	evtapi "github.com/DataDog/datadog-agent/pkg/util/winutil/eventlog/api"
+	eventlog_test "github.com/DataDog/datadog-agent/pkg/util/winutil/eventlog/test"
```

Comment:
I have no idea how this was working before 😮

Reply:
The path is `.../eventlog/api` but if you look at the code the package name is `evtapi`, so it can be used with that name
That's just some Go weirdness 😄

Reply:
I had no idea that was a thing TIL. Thanks for sharing that 😄